src/Mod/Sketcher/Gui/TaskSketcherConstraints.cpp

```diff
@@ -1649,6 +1649,8 @@ bool TaskSketcherConstraints::isConstraintFiltered(QListWidgetItem* item)
                 break;
         }
 
+        visible |= constraint->Name != "" && checkFilterBitset(multiFilterStatus, FilterValue::Named);

```

Comment:
I would recommend using `std::string`' `.empty()` function instead.
```suggestion
        visible |= !constraint->Name.empty() && checkFilterBitset(multiFilterStatus, FilterValue::Named);
```